lib/ui/creators_corner_page.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class CreatorsCornerPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Container(
+        child: const Text(
+          "Creator's Corner Page",
```

Comment:
let's add a TODO comment here linking WP-13

Reply:
I added a TODO here, can you check to see if this is what you wanted me to put in the TODO?